lane.py

```diff
@@ -583,31 +571,20 @@ def set_list(field_name, value, default=None):
         set_list('media', media, Edition.BOOK_MEDIUM)
         set_list('formats', formats)
 
-        # The lane may be restricted to books that are on a list
-        # from a given data source.
-        custom_list_details = self.custom_lists_for_identifier(
-            list_data_source, list_identifier)
-        (self.list_data_source_id,
-         self.list_ids,
-         self.list_featured_works_query) = custom_list_details
-        if not self.list_featured_works_query:
-            # The parent may have featured works from the list that
-            # could be included here.
-            set_from_parent('list_featured_works_query', None)
-        set_from_parent(
-            'list_seen_in_previous_days', list_seen_in_previous_days)
-
-        set_from_parent(
+        self.set_from_parent(
             'subgenre_behavior', subgenre_behavior, self.IN_SUBLANES)
 
-        if self.searchable and (self.list_data_source_id or self.list_ids):
```

Comment:
This is no longer true. I couldn't figure out a check that would make sense to maintain this error.